pkg/virt-handler/vm.go

```diff
@@ -1497,7 +1492,9 @@ func calculatePausedCondition(vmi *v1.VirtualMachineInstance, reason api.StateCh
 			Reason:             "PausedIOError",
 			Message:            "VMI was paused, low-level IO error detected",
 		})
-	default:
+	} else if reason == api.ReasonPausedMigration {
```

Comment:
I'm not sure where this is coming from but I thought libvirt had more expressive events for the convergence step at the end of a migration that it would emit to let the caller ignore the domain being paused. I'll grep around now to see if I'm remembering things correctly it has been a while since I looked at this last.

Reply:
I can totally remove this, is not part of the fix.

Reply:
I can totally remove this, is not part of the fix.

Reply:
https://github.com/libvirt/libvirt/blob/3820a0ce41694eb8b8ae62cfdfd08eb06fb352d4/src/qemu/qemu_migration.h#L94-L114 ah I think it's these enums that I had in mind, I forget how libvirt provides these during a migration but they would provide a meaningful thing to test against if that isn't already happening elsewhere when generating `reason`.

Reply:
@lyarwood I have just remove this